test/integration/editor/index.spec.ts

```diff
@@ -115,3 +115,20 @@ test('code editor auto-complete', async ({ page }) => {
   await page.keyboard.type('textF');
   await expect(page.getByRole('option', { name: 'textField' })).toBeVisible();
 });
+
+test('cancel the status of active when click the white space', async ({ page }) => {
+  const editorModel = new ToolpadEditor(page);
+
+  await editorModel.goToPageById('K7SkzhT');
+
+  await editorModel.waitForOverlay();
+
+  const text = editorModel.appCanvas.getByText('text-foo');
+
+  await clickCenter(page, text);
+  await expect(editorModel.appCanvas.getByTestId('node-hud-tag')).toBeVisible();
+  const element = await editorModel.appCanvas.locator('.Toolpad_HudOverlay');
```

Comment:
Instead of having a locator for the overlay and using `clickCenter` I think that here we should write a `page.mouse.click` with coordinates outside the text element that is selected.

So we can get `text.boundingBox()` for the text element, and then click on `textBoundingBox. x + testBoundingBox.width + 50` and `textBoundingBox.y + testBoundingBox.height + 50`, for example. This way we are sure that we are clicking outside of it.